.drone.yml

```diff
@@ -313,6 +313,170 @@ volumes:
     host:
       path: /var/run/docker.sock
 ---
+name: e2e-ekscluster
+kind: pipeline
+type: docker
+
+depends_on:
+  - qa
+
+clone:
+  depth: 1
+
+trigger:
+  ref:
+    - refs/tags/e2e-full-**
+    - refs/tags/v**
+
+steps:
+- name: run-tests
+  image: ubuntu:latest
```

Comment:
Is it ok to use a plain ubuntu:latest after the new docker limits are in place? We should use one of our images from our own registries https://docs.docker.com/docker-hub/usage/